src/platform/packages/shared/kbn-es-query-server/index.ts

```diff
@@ -19,4 +20,11 @@ export type {
   FilterMeta,
   AggregateQuery,
   Query,
+  SimplifiedFilter,
```

Comment:
```suggestion
  SimpleFilter,
```
Should we align the naming across types instead of using `Simple` and `Simplified` interchangeably (doesn't matter which one between `Simple` and `Simplified` IMO)?

Reply:
Good idea. I've updated the names to be consistent.

TBH, I've also struggled with the `SimpleFilter` name. I've been considering other options such as `DeclarativeFilter`, but naming is hard.